tox.ini

```diff
@@ -3,6 +3,8 @@ envlist = coverage-clean,py{3.9,3.10,3.11,3.12,3.13,py3.9,3.9-nooptionals},cover
 
 [testenv]
 deps =
+    aiohttp
```

Comment:
Similar to twisted it would be good to only install this onto certain versions so that we don't break something core in environments where aiohttp is not present.